docs/usage/resource_management.md

```diff
@@ -0,0 +1,119 @@
+# Resource Requirements
+
+This page covers the steps to create, update and delete resource requests and limits for Argo CD workloads.
+
+The Argo CD Custom Resource allows you to create the workloads with desired resource requests and limits. This is required when a user/admin wish to deploy his Argo CD instance in a namespace that is set with [Resource Quota](https://kubernetes.io/docs/concepts/policy/resource-quotas/).
```

Comment:
@iam-veeramalla Could this be "This is required when a user/admin **wishes** to deploy **their** ArgoCD instance ....."

Reply:
Thanks @tylerauerbeck. Corrected it :)